src/Components/VideoSearchApp.js

```diff
@@ -0,0 +1,56 @@
+import React, { useState } from "react";
+import VideoSearchList from "./VideoSearchList";
+import Player from "./Player";
+import VideoSearch from "./VideoSearch";
+
+import { Container, Row, Col } from "react-bootstrap/";
+
+const VideoSearchApp = () => {
+  const [movies, setMovies] = useState([]);
+  const [movieId, setMovieId] = useState("");
+
+  const searchResultHandler = async (searchText) => {
+    const key = "AIzaSyBsGroGmofzjHBtIG_y4ugaJw8bqSp9eaY";
+    const response = await fetch(

```

Comment:
Also, template literals rather than using the `+` operator to make dynamic strings.